### PR TITLE
API request for webserver endpoint added

### PR DESCRIPTION
**Pull request by kaushalvivek**
## What changes are proposed in this pull request?

(Please fill in changes proposed in this fix)

## How is this patch tested?

(Details)

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components 
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [x] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: Local serving, model deployment tools, spark UDFs
- [ ] `area/server-infra`: MLflow server, JavaScript dev server
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface 
- [ ] `area/uiux`: Front-end, user experience, JavaScript, plotting
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language 
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations
- [x] `integrations/couture-wfo`: Integration with Couture WFO
### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [x] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: mlflow/server/js/src/model-registry/components/ModelVersionView.js [hunk 1]
```diff
@@ -52,20 +54,72 @@ export class ModelVersionViewImpl extends React.Component {
...
     });
+    xhr.onerror = function (e) {
+      console.log(e)
+      this.setState({
+        isModelDeployed: false,
+        modelInferenceAPI: 'Server not reachable.'
+      });
+    }.bind(this);
+    xhr.open('GET', MODEL_DEPLOYMENT_URL + "?runId=" + modelId)
+    xhr.send()
```
Comment (Anubhav2701) [inline mlflow/server/js/src/model-registry/components/ModelVersionView.js]: Refactor this to another file. We want to keep logic in components, specially API calls logic as low as possible.

File: mlflow/server/js/src/model-registry/components/ModelVersionView.js [hunk 2]
```diff
@@ -52,20 +54,72 @@ export class ModelVersionViewImpl extends React.Component {
...
+    xhr.onerror = function (e) {
+      console.log(e)
+      this.setState({
+        isModelDeployed: false,
+        modelInferenceAPI: 'Server not reachable.'
+      });
+    }.bind(this);
+    // send the request
+    xhr.open('POST', MODEL_DEPLOYMENT_URL)
+    xhr.send(JSON.stringify(modelInfo))
```
Comment (Anubhav2701) [inline mlflow/server/js/src/model-registry/components/ModelVersionView.js]: this too

File: mlflow/server/js/src/model-registry/constants.js [hunk 3]
```diff
@@ -76,6 +76,10 @@ export const REGISTERED_MODELS_SEARCH_NAME_FIELD = 'name';
 
 export const REGISTERED_MODELS_SEARCH_TIMESTAMP_FIELD = 'timestamp';
 
+export const MODEL_DEPLOYMENT_URL = 'http://localhost:8080/deploy';
+
+export const MODEL_INFERENCE_API = 'tfserving:8500';
```
Comment (Anubhav2701) [inline mlflow/server/js/src/model-registry/constants.js]: Make these configurable via env vars

**Review by Anubhav2701 — CHANGES_REQUESTED** [review]

File: mlflow/server/js/src/model-registry/components/ModelVersionView.js (hunk 1 above)
Comment (kaushalvivek) [reply mlflow/server/js/src/model-registry/components/ModelVersionView.js]: Done

File: mlflow/server/js/src/model-registry/constants.js (hunk 3 above)
Comment (kaushalvivek) [reply mlflow/server/js/src/model-registry/constants.js]: Done

**Pull request merged**